src/Packetery/Module/Order/LabelPrint.php

```diff
@@ -251,13 +263,23 @@ public function outputLabelsPdf(): void {
 			return;
 		}
 
+		$isClaimPrint = false;
+		foreach ( $orders as $orderId => $order ) {
+			if ( null === $order->getPacketClaimId() || $order->getPacketClaimId() !== $packetIds[ $orderId ] ) {
```

Comment:
Větší smysl by dávalo podmínku obrátit a přepsat výchozí hodnotu.

Reply:
Zatím se nemůže stát, že se budou získávat štítky pro reklamačního asistenta i objednávky zároveň. V takovém případě by bylo potřeba předělat i další kód. Stačí tedy default false a pokud se přijde na to, že jde o reklamačního asistenta, tak nastavit true a použít break. Řádky naopak ubydou.

Reply:
Nemá smysl řešit hypotetickou situaci za cenu nepřehledného kódu.